work/work/mocks/txpool_mock.go

```diff
@@ -1,230 +0,0 @@
-// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Why did you remove this file?

Reply:
I have to add new setter function for mutex based update in blockchain and It also need to regenerate `blockchain_mock` and `txpool_mock`.
I figured it out a duplicated file, work/work/mocks/txpool_mock.go, work/mocks/txpool_mock.go.
Do you know where It can be used?

Reply:
Okay. I agreed this file is duplicated.